conf/app.routes

```diff
@@ -1,4 +1,6 @@
 # microservice specific routes
+->         /govuk-frontend                              govuk.Routes
```

Comment:
Do these two not go in the prod.routes?

Reply:
I was following this README and it said to put it there: https://github.com/hmrc/play-frontend-hmrc/blob/master/README.md